src/snowflake/telemetry/_internal/opentelemetry/exporter/otlp/proto/common/log_encoder.py

```diff
@@ -0,0 +1,22 @@
+from logging import LogRecord
```

Comment:
Please add the copyright header https://github.com/snowflakedb/snowflake-telemetry-python/blob/main/src/snowflake/telemetry/__init__.py#L4

Reply:
Fixed in latest commit.